Car_motor_nano_Shield_inwork.ino

```diff
@@ -45,8 +47,11 @@ void setup() {
   pinMode(trigPin, OUTPUT); 
   pinMode(echoPin, INPUT); 
   
-  myservo.attach(7);  // attaches the servo on pin 7 to the servo object
-  
+  #if 1==NANO
+    myservo.attach(7);  // Use pin 7 in case of NANO
+  #else
+    myservo.attach(5);  // Use pin 5 in case of UNO

```

Comment:
in the code here - this line seems like a comment. not sure if it is right or what it means...
please take a look and see if there is anything to do here or it is OK